.github/workflows/ci-auto.yaml

```diff
@@ -38,3 +38,8 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - run: cd nodeadm && make test-e2e
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
maybe we could do the same thing that vpc-cni proj does with dependency review and govulcheck? https://github.com/aws/amazon-vpc-cni-k8s/blob/b0ff9b904a54a9bec276b7510a92d14ea6ab2d90/.github/workflows/deps.yml

seems like they would need to check the same boxes